synapse/lib/aha.py

```diff
@@ -177,6 +177,23 @@ async def modAhaSvcInfo(self, name, svcinfo):
 
         return await self.cell.modAhaSvcInfo(name, svcinfo)
 
+    async def getAhaSvcMirrors(self, name):
```

Comment:
Does this need a feature flag for exposing this capability, since aha may not be updated alongside of other components?

Reply:
Maybe?  Seems more like a versioning requirement rather than a feature flag item IMO.  Similar issue would exist when calling into the services which may not have the cell API.